.circleci/config.yml

```
@@ -126,7 +126,24 @@ jobs:
       - run:
           name: Deploy docs to gh-pages branch
           command: gh-pages --message "[skip ci] updates" --dist /tmp/_html
-
+  looker-deploy:
+    executor: gcp-cli/google
+    steps:
+      - gcp-cli/setup:
+          use_oidc: true
+          workload_identity_pool_id: GCP_WIP_ID
+          workload_identity_pool_provider_id: GCP_WIP_PROVIDER_ID
+          google_project_number: GCP_PROJECT_ID
+          service_account_email: GCP_SERVICE_ACCOUNT_EMAIL
```

```suggestion
          workload_identity_pool_id: GCPV2_WORKLOAD_IDENTITY_POOL_ID
          workload_identity_pool_provider_id: GCPV2_CIRCLECI_WORKLOAD_IDENTITY_PROVIDER
          google_project_number: GCPV2_WORKLOAD_IDENTITY_POOL_PROJECT_NUMBER
          service_account_email: GCP_SERVICE_ACCOUNT_EMAIL
```
Requires use of context below (see other comment)